.github/workflows/android-perf.yml

```diff
@@ -30,13 +30,47 @@ on:
         description: The list of configs used the benchmark
         required: false
         type: string
+      test_spec:
+        description: The test spec to drive the test on AWS devices
+        required: false
+        type: string
+        default: https://ossci-android.s3.amazonaws.com/executorch/android-llm-device-farm-test-spec.yml
+  workflow_call:
```

Comment:
Why do we need to duplicate the args in `workflow_call:`?

Reply:
It's a GitHub thing, workflow_call and workflow_dispatch are 2 different things and they can accept 2 different set of parameters.  The two are just identical in this case